packages/router/src/grpc/custody/view-transaction-plan/index.ts

```diff
@@ -28,7 +29,7 @@ export const viewTransactionPlan = (
 
   return new TransactionView({
     bodyView: {
-      actionViews: txPlan.actions.map(viewActionPlan(denomMetadataByAssetId)),
+      actionViews: txPlan.actions.map(viewActionPlan(denomMetadataByAssetId, fullViewingKey)),
       fee,
       memoView: {
         memoView: {
```

Comment:
We'll also want to use the FVK to view the return address in the memo plaintext, in the screenshot it shows up as an opaque address but the user should see that it's their address and which account it's for

Reply:
Ah, good catch. I've addressed this in [this commit](https://github.com/penumbra-zone/web/pull/378/commits/39cc3ea2b2dac36d2cd7baba2b1e286c2acb8d36). (I also took the oppt'y to write tests for `getAddressView` in that commit.)

Screenshot:
![image](https://github.com/penumbra-zone/web/assets/1121544/43d31e5c-a02e-40ee-a74f-6653861c8565)